airbyte-integrations/connectors/source-stripe/source_stripe/schemas/payment_methods.json

```diff
@@ -1,5 +1,266 @@
 {
   "$schema": "https://json-schema.org/draft-07/schema#",
   "additionalProperties": true,
-  "$ref": "payment_method.json"
+  "type": ["object", "null"],
+  "properties": {
+    "afterpay_clearpay": {
```

Comment:
I see missing fields like [acss_debit](https://docs.stripe.com/api/payment_methods/object#payment_method_object-acss_debit). Is there a reason for them not being there?

Reply:
copied from the old version of openApi,

BTW, how can we generate resolved openapi -> jsonschema ?

script in tools folder does not work.

Reply:
Unfortunately, I can't answer this one as I've never interacted with the openapi file in stripe... That would be neat. There are two concerns that we would need to validate if we do that:
* Fields are maybe not nullable in the openapi file
* There are some discrepancies between the openapi file and the reality

Reply:
🥲 , updated manually